features/gebruik_in_lopende_tekst.feature

```diff
@@ -206,151 +194,136 @@ Rule: gebruikInLopendeTekst voor een persoon met adellijke titel of predicaat wo
   - Wanneer de persoon een predicaat heeft en het geslacht is gelijk aan "V" (vrouw) en betrokkene gebruikt de naam van de (ex)partner (aanduidingNaamgebruik is ongelijk aan "E")
 
   Abstract Scenario: persoon met adellijke titel heeft aanduidingNaamgebruik "<naamgebruik>"
```

Comment:
Is er een reden waarom in dit scenario ge geslachtsaanduiding niet variabel is gemaakt?
Waarom zijn er niet ook voorbeelden opgenomen met het geslacht 'M' en 'O'?